components/dataViews/TransactionInfo/TxMsgExecuteContractDetails.tsx

```diff
@@ -0,0 +1,64 @@
+import { fromUtf8 } from "@cosmjs/encoding";
+import { MsgExecuteContract } from "cosmjs-types/cosmwasm/wasm/v1/tx";
+import dynamic from "next/dynamic";
+import { useChains } from "../../../context/ChainsContext";
+import { printableCoins } from "../../../lib/displayHelpers";
+import HashView from "../HashView";
+
+const JsonEditor = dynamic(() => import("../../inputs/JsonEditor"), { ssr: false });
+
+interface TxMsgExecuteContractDetailsProps {
+  readonly msgValue: MsgExecuteContract;
+}
+
+const TxMsgExecuteContractDetails = ({ msgValue }: TxMsgExecuteContractDetailsProps) => {
+  const { chain } = useChains();
+
+  return (
+    <>
+      <li>
+        <h3>MsgExecuteContract</h3>
+      </li>
+      <li>
+        <label>Contract:</label>
+        <div title={msgValue.contract}>
+          <HashView hash={msgValue.contract} />
+        </div>
+      </li>
+      <li>
+        <label>Funds:</label>
+        <div>{printableCoins(msgValue.funds, chain)}</div>
+      </li>
+      <li>
+        <JsonEditor readOnly content={{ json: JSON.parse(fromUtf8(msgValue.msg, true)) }} />
```

Comment:
What's the reason for using lossy mode (`true`) here? I think if the contents of the variable is not valid UTF-8, something went wrong and we should throw.